controllers/warp/configReader.go

```diff
@@ -104,35 +119,48 @@ func (reader *ConfigReader) dogusReader(source config.Source) (types.Categories,
 	return reader.createCategories(dogus), nil
 }
 
-func (reader *ConfigReader) getDisabledSupportIdentifiers() ([]string, error) {
-	disabledSupportEntries, err := reader.registry.Get(disableWarpSupportEntriesConfigurationKey)
+func (reader *ConfigReader) readStrings(registryKey string) ([]string, error) {
+	entry, err := reader.registry.Get(registryKey)
+	if err != nil {
+		return []string{}, fmt.Errorf("failed to read configuration entry %s from etcd: %w", registryKey, err)
+	}
+
+	var strings []string
+	err = json.Unmarshal([]byte(entry), &strings)
+	if err != nil {
+		return []string{}, fmt.Errorf("failed to unmarshal etcd key to string slice: %w", err)
+	}
+
+	return strings, nil
+}
+
+func (reader *ConfigReader) readBool(registryKey string) (bool, error) {
+	entry, err := reader.registry.Get(registryKey)
```

Comment:
🚫 **[checkstyle]** <typecheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>reader.registry.Get undefined (type watchConfigurationContext has no field or method Get)

Reply:
That is not true => False-Positive